packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -201,6 +203,15 @@ class Tooltip extends StatefulWidget {
   /// pointer exits the widget.
   final Duration? showDuration;
 
+  /// The mode that will decide when the tooltip will trigger
+  final TooltipTriggerMode? triggerMode;
+
+  /// Whether the tooltip will provide feedback when triggered.
+  ///
+  /// Defaults to true. Will call the appropriate method on `Feedback`
+  /// depending on the trigger mode, such as `Feedback.forLongPress`.
+  final bool? provideTriggerFeedback;
```

Comment:
I am not sure of the motivation for this addition, it 's not mentioned in the linked issue. It does seem similar to the `enableFeedback` in other widgets like `BottomNavigationBar` though. If it is similar to that property, this should probably follow the same pattern as `enableFeedback`.

Reply:
The motivation was for my own personal case, where I didn't want to hear the sound made by the feedback - it sounds a bit weird in mobile imo.

Will follow the pattern.

Reply:
Tried to look at the pattern on `BottomNavigationBar`, but it just seems to pass it along to `InkResponse`.
I renamed the property and changed docs, if that is what you meant?